app/robot-shop/helm/templates/user-deployment.yaml

```diff
@@ -21,12 +21,6 @@ spec:
       - name: user
         image: {{ .Values.image.repo }}/rs-user:{{ .Values.image.version }}
         imagePullPolicy: {{ .Values.image.pullPolicy }}
-        env:
-          # agent networking access
```

Comment:
we should put each of these blocks under a helm flag `instana.enabled: [true | false]`